tuner/examples/punet/mmt.mlir

```diff
@@ -0,0 +1,76 @@
+// RUN: iree-compile --iree-hal-target-backends=rocm --iree-rocm-target-chip=gfx942 \
+// RUN:   --iree-rocm-link-bc=true --iree-rocm-bc-dir=/opt/rocm/amdgcn/bitcode \
+// RUN:   --iree-global-opt-propagate-transposes=true --iree-opt-outer-dim-concat=true \
+// RUN:   --iree-opt-const-eval=false --iree-codegen-gpu-native-math-precision=true --iree-rocm-waves-per-eu=2 \
+// RUN:   --iree-preprocessing-pass-pipeline='builtin.module(iree-preprocessing-transpose-convolution-pipeline)' \
+// RUN:   --iree-codegen-llvmgpu-use-vector-distribution --iree-codegen-transform-dialect-library=config.mlir \
+// RUN:   %s -o %s.vmfb
+
+// To compile to for benchmarking, add:
+//  --iree-flow-export-benchmark-funcs --iree-hal-benchmark-dispatch-repeat-count=1000
+//
+// To benchmark:
+//   for i in {0..4} ; do
+//     iree-benchmark-module --device=rocm://7 --module=%s.vmfb --function="main_${i}_benchmark" --device_allocator=caching \
+//       --batch_size=1000 --benchmark_repetitions=5
+//   done
```

Comment:
Seems weird to pin to device 7. Any particular reason for that? (examples should be _minimal_, so just `--device=hip`)